scrapers/superbemodels.yml

```
@@ -0,0 +1,29 @@
+name: superbemodels
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - superbemodels.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="item-name-model-video item-tags-change"]//h1
+      Date:
+         selector: //span[@class="duration-box duration-box-change"]//span[contains(.,"Release date:")]/em
```

although it seems to work we can simpliify and restrict  it  a bit
```suggestion
        selector: //span[@class="duration-box duration-box-change"][contains(span,"Release date:")]/em
```